### DISCUSSION TITLE
Hide module from module bar based on role.

### DISCUSSION THREAD

**Discussion by NecoHorne** [directus/directus#16738] [Archive]
Hi 

Is there a way to hide specific modules in the module bar based on a specific role? I have went through the docs and can't seem to find anything. I see that I can hide it completely in the project settings but I still want some roles to see some modules. 

For some users I only want them to see a specific custom module. Looked through the discussions and can also not find a definitive answer.

**Comment by rijkvanzanten** [comment]
There's currently no per-role overrides for the module bar

**Comment by rafarje** [comment]
Up up up! 
This would be very helpful!

**Comment by qntmplation** [comment]
This would make a lot of sense to have.

**Comment by domdus** [comment]
Hi, I just released this extension:
https://github.com/domdus/directus-extension-module-permissions

Look for "Module Permissions" in the marketplace.
